src/spartan/batched.rs

```diff
@@ -435,9 +435,10 @@ impl<E: Engine, EE: EvaluationEngineTrait<E>> BatchedRelaxedR1CSSNARKTrait<E>
       )
     }
 
+    let e_r_x = r_x.par_iter().map(|r_x| EqPolynomial::evals_from_points(&r_x)).collect::<Vec<_>>();
```

Comment:
I got a tiny bit confused until I realized it was the `eq` polynomial. Could re rename it to `eq_r_x`?

Reply:
Or even `chi_r_x`